packages/browser/package.json

```diff
@@ -13,17 +13,11 @@
   "scripts": {
     "lint": "eslint . --cache",
     "test": "jest",
-    "version": "node ./scripts/version.js && git add --all",
     "build": "tsc -p tsconfig.main.json && tsc -p tsconfig.module.json",
-    "bundle": "npm run bundle:main && npm run bundle:callback && npm run minify",
```

Comment:
Assuming these can be removed as they're not being run by turborepo. Only regression with this is that callback.js no longer updating its package version.